ext/standard/tests/mail/mail_variation_win.phpt

```diff
@@ -0,0 +1,86 @@
+--TEST--
+Test mail() function : basic functionality (win)
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php require_once __DIR__.'/mail_windows_skipif.inc'; ?>
+--INI--
+SMTP=localhost
+smtp_port=1025
+--FILE--
+<?php
+
+require_once __DIR__.'/mailhog_utils.inc';
+
+$from = ini_get('sendmail_from');
+
+$cases = [
+    [
+        'label' => 'From is not set',
+        'premise' => function () {
+            ini_set('SMTP', 'localhost');
+            ini_set('smtp_port', 1025);
```

Comment:
NIT you don't probably need to repeat this but set it once (you can even use INI section for those two.

Reply:
Thanks, but if you look closely, the values ​​are all slightly different.

Reply:
Ah yeah you right. It looked the same from the initial look.